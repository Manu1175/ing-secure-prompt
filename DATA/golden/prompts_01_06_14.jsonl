{"id": "prompts_01_06_14:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "I found the 2024 Annual Report at https://assets.ing.com/m/6e8ace8ade094690/original/Annual-report-2024.pdf. Can you summarize its main focus?", "expected_scrub": "I found the <YEAR> <DOCUMENT_TYPE> at <DOCUMENT_LINK>. Can you summarize its main focus?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Could you tell me what’s covered in the Pillar 3 Disclosures Q4 2024? Here's the link: https://assets.ing.com/m/5f710bfeff3364d0/original/ING-BE-Pillar-3-Disclosures-Q4-2024.xlsx", "expected_scrub": "Could you tell me what’s covered in the <DOCUMENT_TYPE> <YEAR>? Here's the link: <DOCUMENT_LINK>", "entities": null, "notes": null}
{"id": "prompts_01_06_14:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "I’m reviewing the 2023 Full Year Results. What does the document at http://newsroom.ing.be/strong-results-also-reflect-positively-on-customers-the-economy-and-society say about customer impact?", "expected_scrub": "I’m reviewing the <YEAR> <DOCUMENT_TYPE>. What does the document at <DOCUMENT_LINK> say about customer impact?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the difference between the 2022 and 2023 Pillar 3 Additional Disclosures?", "expected_scrub": "What’s the difference between the <YEAR_1> and <YEAR_2> <DOCUMENT_TYPE>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you compare the 2021 and 2022 Half Year Results in terms of profit growth?", "expected_scrub": "Can you compare the <YEAR_1> and <YEAR_2> <DOCUMENT_TYPE> in terms of profit growth?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the operational status of VM-IAAS-4 deployed on 2023-04-16?", "expected_scrub": "What’s the operational status of <VM_NAME> deployed on <DEPLOYMENT_DATE>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you tell me the cost model and provider for IaaSApp_2?", "expected_scrub": "Can you tell me the <COST_MODEL> and <PROVIDER> for <APPLICATION_NAME>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Which application uses 8 vCPU and 32 GB RAM and was deployed on Google Cloud in 2024?", "expected_scrub": "Which <APPLICATION_NAME> uses <CONFIG> and was deployed on <PROVIDER> in <YEAR>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Is VM-IAAS-8 still active? It was marked as retired in the development environment.", "expected_scrub": "Is <VM_NAME> still active? It was marked as <STATUS> in the <ENVIRONMENT> environment.", "entities": null, "notes": null}
{"id": "prompts_01_06_14:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Which Azure-hosted application has the highest storage capacity?", "expected_scrub": "Which <PROVIDER>-hosted <APPLICATION_NAME> has the highest <STORAGE_CAPACITY>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the last interaction date for Anna Williams, listed as an Insurance Subscriber?", "expected_scrub": "What’s the last interaction date for <ENTITY_NAME>, listed as <RELATIONSHIP_TYPE>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Is Chris Williams still active as an Account Holder under FATCA and KYC?", "expected_scrub": "Is <ENTITY_NAME> still active as <RELATIONSHIP_TYPE> under <REGULATORY_TAGS>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Which customer has a Premium Contract and is linked via API under GDPR and AML?", "expected_scrub": "Which <ENTITY_TYPE> has a <CONTRACT_TYPE> and is linked via <SERVICE_CHANNEL> under <REGULATORY_TAGS>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you list all customers with Digital Consent and active status?", "expected_scrub": "Can you list all <ENTITY_TYPE> with <CONTRACT_TYPE> and <STATUS> status?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the service channel used by Robert Williams for his loan agreement?", "expected_scrub": "What’s the <SERVICE_CHANNEL> used by <ENTITY_NAME> for their <RELATIONSHIP_TYPE>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Is TechNova Solutions still active as an IT Vendor?", "expected_scrub": "Is <ENTITY_NAME> still <STATUS> as an <RELATIONSHIP_TYPE>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Which supplier has a Consulting Agreement and interacts via API?", "expected_scrub": "Which <ENTITY_TYPE> has a <CONTRACT_TYPE> and interacts via <SERVICE_CHANNEL>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Who is the most recent customer to interact under GDPR and KYC?", "expected_scrub": "Who is the most recent <ENTITY_TYPE> to interact under <REGULATORY_TAGS>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Does Laura Lewis have an active insurance subscription?", "expected_scrub": "Does <ENTITY_NAME> have an active <RELATIONSHIP_TYPE>?", "entities": null, "notes": null}
{"id": "prompts_01_06_14:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_06_14.xlsx", "modality": "text", "c_level": null, "raw_text": "Is DataBridge Analytics still under an Annual Contract?", "expected_scrub": "Is <ENTITY_NAME> still under a <CONTRACT_TYPE>?", "entities": null, "notes": null}
