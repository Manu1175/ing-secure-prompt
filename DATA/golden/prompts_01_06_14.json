[
  {
    "text": "I found the 2024 Annual Report at https://assets.ing.com/m/6e8ace8ade094690/original/Annual-report-2024.pdf. Can you summarize its main focus?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Could you tell me what’s covered in the Pillar 3 Disclosures Q4 2024? Here's the link: https://assets.ing.com/m/5f710bfeff3364d0/original/ING-BE-Pillar-3-Disclosures-Q4-2024.xlsx",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "I’m reviewing the 2023 Full Year Results. What does the document at http://newsroom.ing.be/strong-results-also-reflect-positively-on-customers-the-economy-and-society say about customer impact?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What’s the difference between the 2022 and 2023 Pillar 3 Additional Disclosures?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Can you compare the 2021 and 2022 Half Year Results in terms of profit growth?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "What’s the operational status of VM-IAAS-4 deployed on 2023-04-16?",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Can you tell me the cost model and provider for IaaSApp_2?",
    "labels": []
  },
  {
    "text": "Which application uses 8 vCPU and 32 GB RAM and was deployed on Google Cloud in 2024?",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Is VM-IAAS-8 still active? It was marked as retired in the development environment.",
    "labels": []
  },
  {
    "text": "Which Azure-hosted application has the highest storage capacity?",
    "labels": []
  },
  {
    "text": "What’s the last interaction date for Anna Williams, listed as an Insurance Subscriber?",
    "labels": []
  },
  {
    "text": "Is Chris Williams still active as an Account Holder under FATCA and KYC?",
    "labels": []
  },
  {
    "text": "Which customer has a Premium Contract and is linked via API under GDPR and AML?",
    "labels": []
  },
  {
    "text": "Can you list all customers with Digital Consent and active status?",
    "labels": []
  },
  {
    "text": "What’s the service channel used by Robert Williams for his loan agreement?",
    "labels": []
  },
  {
    "text": "Is TechNova Solutions still active as an IT Vendor?",
    "labels": []
  },
  {
    "text": "Which supplier has a Consulting Agreement and interacts via API?",
    "labels": []
  },
  {
    "text": "Who is the most recent customer to interact under GDPR and KYC?",
    "labels": []
  },
  {
    "text": "Does Laura Lewis have an active insurance subscription?",
    "labels": []
  },
  {
    "text": "Is DataBridge Analytics still under an Annual Contract?",
    "labels": []
  }
]